client/src/components/Allocations/AllocationsModals/AllocationsTeamViewModal/AllocationsTeamViewModal.jsx

```diff
@@ -51,51 +81,97 @@ const AllocationsTeamViewModal = ({
       size="lg"
       onClosed={resetCard}
     >
-      <ModalHeader toggle={toggle} charCode="&#xe912;">
-        <span>View Team</span>
-        <div>
+      <ModalHeader className="tab-row" toggle={toggle} charCode="&#xe912;">
+        <Tabs
+          value={selectedTab}
+          onChange={handleTabChange}
+          TabIndicatorProps={{ style: { backgroundColor: 'white' } }}
+        >
+          <Tab
+            label="View Team"
+            className={`tab ${selectedTab === 0 ? 'active' : 'inactive'}`}
+          />
           {isManager && (
-            <Button
-              className="btn btn-sm p-0"
-              color="link"
-              onClick={managementToggle}
-            >
-              Manage Team
-            </Button>
+            <Tab
```

Comment:
This tab has a horizontally scrollable view, probably just want to make the width work without scrolling
![Screen Shot 2022-05-25 at 5 05 18 PM](https://user-images.githubusercontent.com/20326896/170375284-1c88ac70-6ada-429f-921a-4d79e754fd56.png)

Reply:
I've taken a look at the table overflow and scroll. There's a lot of nested markup across components (I get why, not complaining) that each have styles trying to solve this problem, but create a too-many-cooks-in-the-kitchen problem. With some careful style removal (mostly) and addition (minor), I believe I can get this table to behave.

Reply:
@edmondsgarrett fixed the horizontal scroll ([screenshot](https://user-images.githubusercontent.com/62723358/171754165-358c996e-9516-469b-a05e-5be7131d7c37.png)).

I'll try to add some columns widths, to avoid cut-off cells.